src/coursescheduler/temp_json_input/schedule_object_old.json

```diff
@@ -0,0 +1,3328 @@
+{

```

Comment:
Do we need to keep this file if we are only using `schedule_object_old.json` in our code? Seems like needless redundancy.

Reply:
This is being used for a test to ensure backwards (company 1) compatibility

Reply:
How are they accessing it then? Do we want to change it so the debug flag uses this json file for them or do they import our package and go from there?

Reply:
They are using their own inputs AFAIK. I think we should use the new input with the debug flag because it is consistent with the structure we will get from our back-end.

Reply:
So we don't need to keep the old file then?